TELCODOCS-405 updating multi-network policy to address SR-IOV support (#51793)
openshift/openshift-docs: modules/nw-networkpolicy-allow-application-particular-namespace.adoc

// Module included in the following assemblies:
//
// * networking/multiple_networks/configuring-multi-network-policy.adoc
:name: network
:role: admin
ifeval::[{product-version} >= 4.6]
:ovn:
endif::[]
ifeval::["{context}" == "configuring-multi-network-policy"]
:multi:
:name: multi-network
:role: cluster-admin
endif::[]

:_content-type: PROCEDURE
[id="nw-networkpolicy-allow-traffic-from-a-namespace_{context}"]
= Creating a {name} policy allowing traffic to an application from a namespace

[NOTE]
====
If you log in with a user with the `cluster-admin` role, then you can create a network policy in any namespace in the cluster.
====

> Follow this procedure to configure a policy that allows traffic to a pod with the label `app=web` from a particular namespace. You might want to do this to:

Reviewer: Something like: 
```suggestion
You can configure a policy that allows traffic to a pod with the label `app=web` from a particular namespace to complete the following tasks:
```
